docs/docs/pods.md

```diff
@@ -0,0 +1,583 @@
+---
+title: Pods
+---
+
+# Pods
+
+Marathon supports the creation and management of pods. Pods enable you to share storage, networking, and other resources among a group of applications on a single agent and address them as one group rather than as separate applications, similar to a virtual machine. Pods allow quick, convenient coordination between applications that need to work together, for instance a database and web server that make up a content management system.
+
+Currently, Marathon pods can only be created and administered via the `/v2/pods/` endpoint of the REST API, not via the web interface.
+
+# Features
+
+- Co-located containers.
+- Pod-level resource isolation.
+- Sandbox and ephemeral volumes that are shared among pod containers.
+- Configurable sandbox mount point for each container.
+- Pod-level health checks.
+
+# Quick Start
+
+1. Run the following REST call, substituting your IP and port for `<ip>` and `<port>`:
+
+    ```bash
+    $ http POST <ip>:<port>/v2/pods <<EOF
+    > {
+    >   "id": "/simplepod",
+    >   "scaling": { "kind": "fixed", "instances": 1 },
+    >   "containers": [
+    >     {
+    >       "name": "sleep1",
+    >       "exec": { "command": { "shell": "sleep 1000" } },
+    >       "resources": { "cpus": 0.1, "mem": 32 }
+    >     }
+    >   ],
+    >   "networks": [ {"mode": "host"} ]
+    > }
+    > EOF
+    ```
+
+1. Verify the status of your new pod:
+
+    ```bash
+    http GET <ip>:<port>/v2/pods/simplepod::status
+    ```
+
+1. Delete your pod:
+
+    ```bash
+    http DELETE <ip>:<port>/v2/pods/simplepod
+    ```
+
+# Technical Overview
+
+Pods are are members of Groups.* A pod instance’s containers are launched together, atomically, via the Mesos LAUNCH_GROUP call. Containers in pods share networking namespace and ephemeral volumes.
```

Comment:
pods are not members of a group... a pod defines a group... the apps or containers are members